code/__DEFINES/bodyparts.dm

```diff
@@ -0,0 +1 @@
+#define IS_ORGANIC_LIMB(A) (A.bodytype & BODYTYPE_ORGANIC)
```

Comment:
shouldn't this go in `HELPERS`

Reply:
It's a macro not a helper proc so I believe it belongs here.

Reply:
Helpers are often macros, and SuperNovaa is right